thiscovery_admin/projects/migrations/0083_auto_20230412_1154.py

```
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.18 on 2023-04-12 11:54
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0082_usergroup_project'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='userlistmembership',
+            name='anon_project_specific_user',
+        ),
+        migrations.AddField(
+            model_name='userlistmembership',
+            name='aps_or_real_user_id',
+            field=models.UUIDField(default=uuid.uuid4),
```

do we need a default here?  The creating process should always provide it and if it doesn't then we should throw an error, not default it.  So I'd suggest omitting default, but leaving as non null

---

"You are trying to add a non-nullable field 'aps_or_real_user_id' to userlistmembership without a default; we can't do that (the database needs something to populate existing rows)."